README.md

```diff
@@ -240,7 +237,10 @@ $ s2i build git://github.com/bparees/openshift-jee-sample openshift/wildfly-100-
 $ docker run --rm -i -p :8080 -t test-jee-app
 ```
 
-Interested in more advanced `s2i` usage? See [here](https://github.com/openshift/source-to-image/blob/master/docs/cli.md)
-for detailed descriptions of the different CLI commands with examples.
+Want to know more? Read the following resources:
 
-Running into some issues and need some advice debugging?  Peruse [here](https://github.com/openshift/source-to-image/blob/master/docs/debugging-s2i.md) for some tips.
+* [description of the different S2I commands with examples](docs/cli.md)
+* [tips for using builder images](docs/user_guide.md)
```

Comment:
i think tips is the wrong word.  it's more like "Instructions for using builder images"  (at least that should be the goal)